cmake/windows-dpcpp-toolchain.cmake

```diff
@@ -12,6 +12,8 @@
 #
 ##===----------------------------------------------------------------------===##
 
+# This toolchain file should ONLY be included on Windows for CMAKE_CXX_COMPILER=dpcpp, icpx
```

Comment:
What do you think about adding a warning when the toolchain is included with a wrong compiler?

Reply:
Yes, I will add this.  I think perhaps an error is warranted rather than a warning.

Reply:
With the removal of the toolchain file, this is no longer applicable